src/puppetlabs/puppetdb/http/server.clj

```diff
@@ -11,18 +11,6 @@
             [net.cgrand.moustache :refer [app]]
             [ring.util.response :as rr]))
 
-(defn deprecated-app
```

Comment:
I've re-used experimental-warning elsewhere, so I moved these both to http.clj for general use.